portfolio/src/main/webapp/Controller.js

```diff
@@ -0,0 +1,77 @@
+import CommentHandler from '/CommentHandler.js';
+import UserHandler from '/UserHandler.js';
+
+export default class Controller {
+
+  constructor(document) {
+    this.username = undefined;
+    this.loggedIn = false;
+    this.document = document;
+  }
+
+  /** Shows the change name form*/
+  openNameForm() {
+    this.document.getElementById('name-form').classList.remove('hidden-elem');
+  }
+
+  /** Hides the change name form*/
+  closeNameForm() {
+    this.document.getElementById('name-form').classList.add('hidden-elem');
+  }
+
+  /**
+   * Loads all comments from backend.
+   * This function is called when the page is loaded.
+   * @param {HTML div object} commentBox The div in document with id "comment-box"
+   * @param {HTML template} commentCell The template for one comment
+   */
+  static async loadComments(commentBox, commentCell) {
+    try {
+      let commentArray = await CommentHandler.loadComments();
```

Comment:
To me the controller is actually the adapter between view and model, so only the controller can interact with the view. The service modules can get data from the server but they cannot directly change the view.
This is design acceptable?